src/app/lib/utilities/variantHandler/index.js

```diff
@@ -21,3 +21,16 @@ export const getVariant = ({ service, variant }) => {
 
   return 'default';
 };
+
+export const getOtherVariant = (service, variant) => {
```

Comment:
try to move this function to be with scriptLink, and refactor function to not use `serviceWithVariants`.